src/client/components/app/Transactions/EnableCollateralAssetTx.tsx

```diff
@@ -78,17 +78,13 @@ export const EnableCollateralAssetTx: FC<EnableCollateralAssetTxProps> = (props)
   const [transactionCompleted, setTransactionCompleted] = useState(0);
   const [transactionApproved, setTransactionApproved] = useState(true);
   const [transactionLoading, setLoading] = useState(false);
-  const [selectedCollateralAssetAddress, setSelectedCollateralAssetAddress] = useState('');
+  const [selectedCollateralAssetAddress, setSelectedCollateralAssetAddress] = useState(collateralOptions[0].address);
```

Comment:
can you use `state.collateral.selectSelectedCollateralAsset` instead of local state plz.

Reply:
@kibagateaux I did not have time to fix this minor issue. I'll roll this into the next PR. Can you please merge? Thanks!